notebook_analyzer/connector/get_data.py

```diff
@@ -173,9 +193,8 @@ def cells(self):
         return self._cells
 
     def get_script_source(self):
+        path = os.path.abspath(self._metadata['name'])
 
-        route = os.path.abspath('/Users/konstantingrotov/Documents/datasets/20kk_dataset') + '/'
-        path = route + self._metadata['name']
         with open(path, 'r', encoding="utf-8") as f:
             source = f.read()
         if len(source) > 20_000:
```

Comment:
Magic number, also great to see in config. Next PR

Reply:
It seems to me that it's better not to insert it into the config, because it is more about boundaries of lib, and not the configuration of scripts processing. IDK, I added it to avoid handling big scripts at the beginning, when I had much more AST creations.